packages/runtime/src/message-type.ts

```diff
@@ -67,7 +67,14 @@ export class MessageType<T extends object> implements IMessageType<T> {
         this.typeName = name;
         this.fields = fields.map(normalizeFieldInfo);
         this.options = options ?? {};
-        this.messagePrototype = Object.defineProperty({}, MESSAGE_TYPE, { value: this });
+        this.messagePrototype = this.messagePrototype = Object.defineProperty(
+            Object.create(
+              null,
+              Object.getOwnPropertyDescriptors(Object.getPrototypeOf({}))
+            ),
+            MESSAGE_TYPE,
+            { value: this },
+        );
```

Comment:
We should probably store a reference to the object property descriptors just once outside the class:
```ts
const baseDescriptors = Object.getOwnPropertyDescriptors(Object.getPrototypeOf({}));
```

We can also simplify this a bit
```suggestion
        this.messagePrototype = Object.create(null, { ...baseDescriptors, [MESSAGE_TYPE]: { value: this } });
```

Reply:
@jcready done!